docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/ra1n-chromeos.md

```diff
@@ -0,0 +1,80 @@
+---
+lang: en_US
+title: Installing Odysseyra1n (ChromeOS)
+description: Guide to installing Odysseyra1n on ChromeOS
+permalink: /installing-odysseyra1n-chromeos
+redirect_from:
+  - /installing-checkra1n/chromeos
+  - /installing-checkra1n-chromeos
+extra_contributors:
+  - DhinakG
+  - Tanbeer191
+  - TheHacker894
+  - zachary7829
+  - stekc
+  - WhitetailAni
+---
+
+::: warning
+
+If you are using a computer with an AMD Ryzen CPU, you will likely run into issues. If you do run into issues, you should use a Mac or a computer with an Intel CPU to follow this guide.
+
+:::
+
+::: warning
+
+If you have an Apple TV HD and are using a USB-C to UsB-C to do this process, you may run into issues entering into DFU mode
+
+If you do have issues, get a USB-A to USB-C cable and, if necessary, also get a USB-C to USB-A adapter.
+
+:::
+
+::: danger
+
+This is a highly experimental guide page, there may be things not directly mentioned that need to be done in order to follow this. As a result, we highly recommend getting another type of computer instead if possible.
+
+:::
+
+::: tip
+
+You need to have enabled developer mode on your ChromeOS device to follow this
```

Comment:
Should be a period at the end of this sentence, needs fixing on original chromeOS guide too